instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './CommentSection.css';
+import CommentInput from './CommentInput';
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: props.comments,
+            comment: ''
+        }
+    }
+
+    commentHandler = e => {
+        this.setState({ comment: e.target.value });
+      };
+
+    addNewComment = e => {
+        e.preventDefault();
+        const newComment = { text: this.state.comment, username: (localStorage.getItem('user')) };
+        const comments = this.state.comments;
+        comments.push(newComment);
+        this.setState({ comments, comment: '' });
+    };
+    
+    render () {
+        return (
+            <div>
+                {this.state.comments.map(element => (
+                    <div className='comment'>
```

Comment:
Need a key prop when rendering a list of data
id's are preferred to be used as a key but since our dummy-data doesn't provide an id, index will do
```suggestion
                    {this.state.comments.map((element, index) => (
                    <div className='comment' key={index}>
```